clients/gateway/gateway.go

```diff
@@ -85,12 +93,12 @@ func (c *Client) AddInvokeTransaction(txn json.RawMessage) (json.RawMessage, err
 	}
 	defer body.Close()
 
-	invokeRes, readErr := io.ReadAll(body)
+	deployRes, readErr := io.ReadAll(body)
```

Comment:
Since this is used for invoke and deploy, can we use a generic name like `res`?

Reply:
Yes, good point!